mir_eval/separation.py

```diff
@@ -422,6 +703,33 @@ def _bss_source_crit(s_true, e_spat, e_interf, e_artif):
     return (sdr, sir, sar)
 
 
+def _bss_image_crit(s_true, e_spat, e_interf, e_artif):
+    """Measurement of the separation quality for a given image in terms of
+    filtered true source, spatial error, interference and artifacts.
+
+    Parameters
```

Comment:
Sorry, I meant just the parameters.  The docstring description part is still useful!